apps/civil/civil-service/demo.yaml

```diff
@@ -8,7 +8,7 @@ spec:
     java:
       ingressClass: traefik-private
       disableIngressClassAnnotation: true
-      image: hmctspublic.azurecr.io/civil/service:prod-118873f-20221219124336 #{"$imagepolicy": "flux-system:civil-service"}
+      image: hmctspublic.azurecr.io/civil/service:pr-1488-cb71981-20221213113911 #{"$imagepolicy": "flux-system:civil-service"}
```

Comment:
imagepolicy value needs to be 
`flux-system:demo-civil-service`

Reply:
Changed flux system to demo-civil-service.